package.json

```diff
@@ -34,6 +34,9 @@
     },
     "linux": {
       "target": "deb"
+    },
+    "win": {
+      "target": "zip"
```

Comment:
nsis was giving an error for me, unfortunately - should dig into that soonish, but zip works so lets get it out.